src/api/auth.ts

```diff
@@ -118,6 +118,12 @@ export function getTeamUsers(team: string, organization: string, auth: any): any
   return users;
 }
 
+export function checkPackageName(pkgName: string): boolean {
+  let regexStart = /^[a-z0-9~()'!*:@,;+?-]{1}/g;
```

Comment:
this regexes cannot work, you are allowing all of `~()'!*:@,;+?-` characters in a package name. where did you get this? help yourself with http://regexr.com/.